code/game/gamemodes/gang/vigilante.dm

```diff
@@ -0,0 +1,203 @@
+/obj/item/device/vigilante_tool
+	name = "Vigilant's Companion"
+	desc = "A reverse-engineered gang tool designed by Nanotrasen to encourage crew to resist gang activity."
+	icon_state = "gangtool-white"
+	item_state = "walkietalkie"
+	w_class = WEIGHT_CLASS_TINY
+	origin_tech = "programming=5;bluespace=2"
+	var/points = 0
+	var/list/tags = list()
```

Comment:
Ive never made an entire list "lazy" or whatever and I can't see any examples that make it clear.